swap_meet/decor.py

```diff
@@ -1,2 +1,14 @@
-class Decor:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Decor(Item):
+    def __init__(self, id=None, condition=0, width=0, length=0):
+        super().__init__(id, condition)
+
+        self.width, self.length = [width, length]
```

Comment:
No need to put this inside of a list, you could simply do:
```python
self.width, self.length = width, length
``` 
However, I will say I'd opt for putting them on separate lines to be consistent and in alignment with the rest of the coding style of the project.